core/main/src/broker/http_broker.rs

```diff
@@ -24,57 +24,80 @@ use ripple_sdk::{
     tokio::{self, sync::mpsc},
     utils::error::RippleError,
 };
+use serde_json::Value;
 
 use super::endpoint_broker::{
     BrokerCallback, BrokerCleaner, BrokerConnectRequest, BrokerOutputForwarder, BrokerRequest,
     BrokerSender, EndpointBroker, EndpointBrokerState, BROKER_CHANNEL_BUFFER_SIZE,
 };
-use crate::state::platform_state::PlatformState;
+use crate::{
+    broker::rules_engine::{jq_compile, RuleTransformType},
+    state::platform_state::PlatformState,
+};
 use tokio_tungstenite::tungstenite::http::uri::InvalidUri;
 
 pub struct HttpBroker {
     sender: BrokerSender,
     cleaner: BrokerCleaner,
 }
-/*
-
-*/
 
 async fn send_http_request(
     client: &Client<HttpConnector>,
-    method: Method,
     uri: &Uri,
-    path: &str,
+    broker_request: BrokerRequest,
```

Comment:
This adds POST support to `HttpBroker`, wherein if the rule has a request transform it is assumed to be the POST request body.